vars/commitPragmaTrusted.groovy

```diff
@@ -39,13 +38,20 @@ String call(String name, String def_val = null) {
      *           }
      *       }
      */
-    if (env.COMMIT_MESSAGE && env.COMMIT_MESSAGE != "") {
+    if (env.COMMIT_MESSAGE && env.COMMIT_MESSAGE != '') {
         commit_message = env.COMMIT_MESSAGE
     } else {
-        commit_message = sh(script: 'git show -s --format=%B',
+        String cmd = 'if [ -n "$GIT_CHECKOUT_DIR" ] && ' +
+                     '''[ -d "$GIT_CHECKOUT_DIR" ]; then
+                          cd "$GIT_CHECKOUT_DIR"
+                        fi
```

Comment:
While it's not bad to have this here, you mentioned that it's not really going to work for us per your comment elsewhere:

> (01:05:38 PM) John Malmberg: Brian: Github Multibranch Pipelines do not set the "GIT_CHECKOUT_DIR" environment variable.

Is the above still true and this change while useful maybe somewhere won't help us?